examples/local_ratelimit/ratelimit-envoy.yaml

```diff
@@ -95,6 +99,14 @@ static_resources:
                     key: x-local-rate-limit
                     value: 'true'
               local_rate_limit_per_downstream_connection: false
+              descriptors:
+              - entries:
```

Comment:
If we want to show how descriptors work, two descriptors with different token bucket settings make more sense.

Reply:
thanks, PTAL again.